.fixtures.yml

```diff
@@ -2,11 +2,11 @@ fixtures:
   repositories:
     stdlib:
       repo: 'https://github.com/puppetlabs/puppetlabs-stdlib.git'
-      ref: '5.2.0'
+      ref: 'v7.1.0'
```

Comment:
Are we using some functionality that requires more recent versions of stdlib (and other fixtures) or are this just a version bump?

Reply:
This is just a version bump.
If we want to use stdlib::Yes_no we need v6.1.0. Otherwise we should be good with 5.2.0.